test/e2e/src/step-definitions/launchpad.steps.ts

```diff
@@ -0,0 +1,106 @@
+import { When, Then, Given, DataTable } from '@cucumber/cucumber'
+import supertest from 'supertest'
+import assert from 'assert'
+
+const request = supertest('https://development--spacex-rest-api.netlify.app/')
+
+let launchpadId: null = null  
```

Comment:
Тука оваа варијабла после се мене не е само од тип null нели? Може да ставеш вака

```suggestion
let launchpadId: null | string = null  
```